test/test_common/utility.h

```diff
@@ -527,14 +527,44 @@ ApiPtr createApiForTest(Event::TimeSystem& time_system);
 ApiPtr createApiForTest(Stats::Store& stat_store, Event::TimeSystem& time_system);
 } // namespace Api
 
-MATCHER_P(HeaderMapEqualIgnoreOrder, rhs, "") {
-  *result_listener << *rhs << " is not equal to " << *arg;
-  return TestUtility::headerMapEqualIgnoreOrder(*arg, *rhs);
+MATCHER_P(HeaderMapEqualIgnoreOrder, expected, "") {
+  bool equal = TestUtility::headerMapEqualIgnoreOrder(*arg, *expected);
```

Comment:
nit: const here and for the other matchers

Reply:
Done.